File: Flix.Backend/Flix.Services/Database/ReportStatus.cs
namespace Flix.Services.Database
{
    /// <summary>
    /// Status of a report raised by a user (either about a movie's details
    /// or about another user's conduct/content).
    /// </summary>
    public enum ReportStatus
    {
        Open = 0,
        Resolved = 1,
        Dismissed = 2
    }
}

File: Flix.UI/flix_desktop/lib/enums/report_status.dart
import 'package:json_annotation/json_annotation.dart';

enum ReportStatus {
  @JsonValue(0)
  open,
  @JsonValue(1)
  resolved,
  @JsonValue(2)
  dismissed
}

String getReportStatus(ReportStatus status) {
  if (status == ReportStatus.open) {
    return "Open";
  } else if (status == ReportStatus.resolved) {
    return "Resolved";
  } else {
    return "Dismissed";
  }
}
